src/main/kotlin/uk/gov/dwp/dataworks/services/ExistingUserServiceCheck.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dwp.dataworks.services
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.ecs.EcsClient
+
+@Service
+class ExistingUserServiceCheck{
+    @Autowired
+    private lateinit var ecsDescribeServicesCall: EcsDescribeServicesCall
+    val configService = ConfigurationService()
```

Comment:
This should be autowired in, instead of created explicitly

Reply:
Good call, I am unsure how i missed that